lib/dor/models/describable.rb

```diff
@@ -275,15 +275,11 @@ def set_desc_metadata_using_label(force = false)
     end
 
     def self.get_collection_title(obj)
-      xml = obj.descMetadata.ng_xml
-      title = ''
-      title_node = xml.at_xpath('//mods:mods/mods:titleInfo/mods:title', 'mods' => 'http://www.loc.gov/mods/v3')
-      if title_node
-        title = title_node.content
-        subtitle = xml.at_xpath('//mods:mods/mods:titleInfo/mods:subTitle', 'mods' => 'http://www.loc.gov/mods/v3')
-        title += " (#{subtitle.content})" if subtitle
-      end
-      title
+      obj.full_title
+    end
+
+    def full_title
+      stanford_mods.sw_title_display
```

Comment:
do we care that this is functionally different? that is, stanford_mods adds the part number/name to the title, and puts the subtitle in but not in parens. I'm guessing that stanford_mods is more correct anyway.